game/scripts/vscripts/Sections/Chapter4/SectionCommunication.ts

```diff
@@ -8,23 +9,22 @@ const sectionName: SectionName = SectionName.Chapter4_Communication;
 
 let graph: tg.TutorialStep | undefined = undefined
 
-/**
- * Describes the state we want the game to be in before this section is executed. The game will try to make the state match this required state.
- */
 const requiredState: RequiredState = {
+    requireSlacksGolem: true,
+    requireSunsfanGolem: true,
```

Comment:
Requires the golems, but doesn't matter where they are? Is this just to keep them alive so they can speak?

Maybe put them somewhere around where everything's happening.

Reply:
The default position is supposed to be the ghost position for them